gen_topo.sh

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env sh
+#PBS -P tm70
+#PBS -q normal
+#PBS -l walltime=4:00:00,mem=4GB
+#PBS -l wd
+#PBS -lstorage=gdata/hh5+gdata/ik11+gdata/tm70
+
+module use /g/data/hh5/public/modules
+module load conda/analysis3
+
+# Build domain-tools
+./build.sh
+
+set -x
+set -e
+
+cp -L --preserve=timestamps /g/data/tm70/ek4684/New_grid_input_files_025deg_75zlevels/ocean_hgrid.nc .
+cp -L --preserve=timestamps /g/data/tm70/ek4684/New_grid_input_files_025deg_75zlevels/ocean_vgrid.nc .
+ln -sf /g/data/ik11/inputs/GEBCO_2024/GEBCO_2024.nc ./
+
+# Convert double precision vgrid to single
+./domain-tools/bin/float_vgrid --vgrid ocean_vgrid.nc --vgrid_type mom6
```

Comment:
The attributes of this file are kind of weird, see the Kial Stewart and Russ Fielder references ? ...

```
ncdump -hs ocean_vgrid.nc 
netcdf ocean_vgrid {
dimensions:
        nzv = 76 ;
variables:
        double zeta(nzv) ;
                zeta:units = "meters" ;
                zeta:standard_name = "vertical_grid_vertex" ;
                zeta:long_name = "vgrid" ;
                zeta:author = "Kial Stewart" ;
                zeta:_Storage = "contiguous" ;
                zeta:_Endianness = "little" ;

// global attributes:
                :original_file = "ocean_vgrid.nc" ;
                :history = "Sun Nov 25 13:35:44 2024 +1100: float_vgrid --vgrid ocean_vgrid.nc --vgrid_type mom6\nThu Sep 19 17:04:43 2024: ncks -O -d nzv,,,2 /g/data/ik11/inputs/access-om2/input_20201102/mom_01deg/ocean_vgrid.nc ocean_vgrid_75.nc\nMon Jul 24 14:14:08 2017: ncatted -a NOTE,global,c,c,Modified by Russ Fiedler. Suitable for double and single precision arithmetic ocean_vgrid.nc" ;
                :_NCProperties = "version=2,netcdf=4.7.3,hdf5=1.10.5," ;
                :_SuperblockVersion = 0 ;
                :_IsNetcdf4 = 1 ;
                :_Format = "netCDF-4" ;

Reply:
The attributes of these files were not generated by `float_vgrid`, it just converts the double to single precision

Reply:
Where does the original file come from? How is it made ?